src/test/response_cache_test.cc

```diff
@@ -182,6 +182,17 @@ InferenceResponse::InferenceResponse(
   // Skip allocator logic / references in unit test
 }
 
+std::ostream&
+operator<<(std::ostream& out, const InferenceResponse& response)
+{
+  out << "[0x" << std::addressof(response) << "] "
+      << "response id: " << response.Id() << std::endl;
+
+  out << "status:" << response.ResponseStatus().AsString() << std::endl;
+
+  return out;
+}
```

Comment:
What was this change needed for?

Reply:
Seems like newer GTest will try to use this outstream operator, so need to provide implementation.